### DISCUSSION TITLE
Terminate Workflow block

### DISCUSSION THREAD

**Discussion by edwyn-amador** [AutomaApp/automa#731] [Ideas]
I've created my workflows to work in a modular way along with the "When visit a website" trigger. At this point I need a block to terminate specific workflows on demand, this is very crucial for my use case in order to make my workflows work as intended, is this something you can implement, please? Thanks

**Comment by mzaini30** [comment]
You can terminate with this JS code block:

```javascript
throw new Error
```

**Reply by edwyn-amador** [reply]
Thanks for answering but this is not the same as the terminate workflow block I suggested, what you suggested would only work in the current workflow plus also requires the workflow to be executing that specific JS block, what I need is to be able to terminate a workflow even from a different workflow by selecting the desired workflow to terminate from a drop down list and of course no matter what operation or block that workflow is running just stop it once that terminate workflow block runs

**Reply by Delusionnnn** [reply]
Workflow State > Stop Specific Workflows

**Comment by edwyn-amador** [comment]
> Workflow State > Stop Specific Workflows

@Delusionnnn I know, thanks, this wasn't available before, this was introduced because of my feature request. I also request it on Discord

**Reply by Delusionnnn** [reply]
Whats the Discord? Need to able to request things too.
